django-stubs/contrib/admin/options.pyi

```diff
@@ -33,6 +33,7 @@ from django.urls.resolvers import URLPattern
 from django.utils.datastructures import _ListOrTuple
 from django.utils.functional import _StrOrPromise
 from django.utils.safestring import SafeString
+from template.context import _ContextKeys
```

Comment:
```suggestion
from django.template.context import _ContextKeys
```

Reply:
Thanks for the feedback. 👍 

Hopefully I've captured them in the required style now 🤞